apis/apis.json

```diff
@@ -161,8 +161,8 @@
     "id": "Google.Cloud.Logging.Log4Net",
     "version": "1.0.0-beta11",
     "type": "other",
-    "targetFrameworks": "net45",
-    "testTargetFrameworks": "net452",
+    "targetFrameworks": "netstandard1.5;net45"
+    "testTargetFrameworks": "netcoreapp1.0;net452"
```

Comment:
No, you either need the lines as per your commit, or change the commit to delete the lines entirely. Basically, the previously-present lines were there to explicitly state targets which *differed* from the default generated by the tool - the new targets *are* the default, so they can either be stated explicitly or not stated at all.

It's probably easiest just to leave it as it is for now though - I can remove it later one with no changes elsewhere.